test/evmc/README.md

```diff
@@ -1,5 +1,10 @@
 # EVMC
 
-This is an import of [EVMC](https://github.com/ethereum/evmc) version [10.1.0](https://github.com/ethereum/evmc/releases/tag/v10.1.0).
+This is an import of [EVMC](https://github.com/ethereum/evmc) version [11.0.0](https://github.com/ethereum/evmc/releases/tag/v11.0.0).
 
-Important: The `MockedAccount.storage` is changed to a `map` from `unordered_map` as ordering is important for fuzzing.
+Steps when upgrading:
+- Copy all from [include/evmc](https://github.com/ethereum/evmc/tree/master/include/evmc) to [test/evmc](https://github.com/ethereum/solidity/tree/develop/test/evmc)
+    - Note that you should delete (or not copy in the first place) `tooling.hpp` and `instructions.h`.
+- Copy [`loader.c`](https://github.com/ethereum/evmc/blob/master/lib/loader/loader.c) to [test/evmc](https://github.com/ethereum/solidity/tree/develop/test/evmc)
+- `MockedAccount.storage` in `mocked_host.hpp` should be changed to a `map` from `unordered_map` as ordering is important for fuzzing. You'll also need to include `<map>`.
+    See [PR #11094](https://github.com/ethereum/solidity/pull/11094) for more details.
```

Comment:
I fixed the link. Originally it looked like this, which is not how links work in markdown (see c53afaec35433ed53788f8c22a5f5ba4e5edf8af):
```markdown
[commit] for more details (https://github.com/ethereum/solidity/commit/788eb028255dc31aa108ac300dca52ad064fabf0)
```

While at it I thought I'd replace it with the link to the PR, since that gives more context.